agrona/src/main/java/org/agrona/concurrent/status/AtomicCounter.java

```diff
@@ -292,11 +347,32 @@ public long getAndAdd(final long increment)
 
     /**
      * Add an increment to the counter with ordered store semantics.
+     * <p>
+     * This method is identical to {@link #getAndAddRelease(long)} and that method should be used instead.
      *
      * @param increment to be added with ordered store semantics.
      * @return the previous value of the counter
      */
     public long getAndAddOrdered(final long increment)
+    {
+        return getAndAddRelease(increment);
+    }
+
+    /**
+     * Adds an increment to the counter non atomically.
+     * <p>
+     * This method is not atomic; it can suffer from lost-updates due to race conditions.
+     * <p>
+     * The load has plain memory semantics and the store has release memory semantics.
+     * <p>
+     * The typical use-case is when there is one mutator thread, that calls this method, and one or more reader
+     * threads.
+     *
+     * @param increment to be added
+     * @return the previous value of the counter
+     * @since 2.1.0
+     */
+    public long getAndAddRelease(final long increment)
```

Comment:
Should call `org.agrona.UnsafeApi#getAndAddLongRelease` to ensure that the operation is atomic.